### PR TITLE
update eslint version

### PR DESCRIPTION
**Pull request by mustafah15**

### CODE REVIEW

**Comment by mustafah15** [comment]
[fixed ch4337]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (bahaagalal) [inline package.json]: Please adjust repository docker, gitignore, circle ci & other init files.

File: .gitignore [hunk 2]
```diff
@@ -28,15 +20,20 @@ node_modules
 
 # Optional npm cache directory
 .npm
+package-lock.json
```
Comment (bahaagalal) [inline .gitignore]: remove `package-lock.json`

File: Dockerfile [hunk 3]
```diff
@@ -5,4 +5,4 @@ ADD . /var/code/
 WORKDIR /var/code/
```
Comment (bahaagalal) [inline Dockerfile]: `WORKDIR` comes first before `ADD`

File: Dockerfile [hunk 4]
```diff
@@ -5,4 +5,4 @@ ADD . /var/code/
...
 RUN mv .npmrc-example .npmrc
```
Comment (bahaagalal) [inline Dockerfile]: should be `COPY ./.npmrc-example ./.npmrc`

File: circle.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+machine:
```
Comment (bahaagalal) [inline circle.yml]: Missing `circle-ci` badge on `readme` file.

File: circle.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    node:
+        version: 6.3.1
```
Comment (bahaagalal) [inline circle.yml]: `6.10.0`

File: circle.yml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+checkout:
+    post:
+        - mv ./.npmrc-example ./.npmrc
+
```
Comment (bahaagalal) [inline circle.yml]: remove this empty line as well as the empty line at line 11.

File: docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  node:
+    tty: true
+    stdin_open: true
+    container_name: eslint-config-crowdanalyzer
```
Comment (bahaagalal) [inline docker-compose.yml]: missing 
```
env_file:
  - ./.env
```

File: package.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
...
   "name": "eslint-config-crowdanalyzer",
-  "version": "1.4.0",
+  "version": "1.4.1",
   "license": "UNLICENSED",
   "author": "CrowdAnalyzer <dev@crowdanalyzer.com>",
   "description": "Crowd Analyzer eslint configuration",
```
Comment (bahaagalal) [inline package.json]: The order of fields after description is `repository`, `engines`, `contributors`, `main`, `scripts`, `devDependencies` & then `peerDependencies`

File: package.json [hunk 10]
```diff
@@ -28,9 +32,9 @@
   },
```
Comment (bahaagalal) [inline package.json]: `docker` `script` should be `docker-compose build --no-cache && docker-compose run node`

File: startup.sh [hunk 11]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Copying .npmrc-example into .npmrc"
+cp /var/code/.npmrc-example /var/code/.npmrc;
```
Comment (bahaagalal) [inline startup.sh]: remove semicolon at the end of line.

File: Dockerfile [hunk 12]
```diff
@@ -5,4 +5,4 @@ ADD . /var/code/
...
 RUN npm install
-ENTRYPOINT ["npm", "run", "lint"]
+ENTRYPOINT ./startup.sh
```
Comment (bahaagalal) [inline Dockerfile]: missing `.dockerignore` file.

**Pull request merged**